models/operationoutcome.go

```diff
@@ -0,0 +1,27 @@
+package models
```

Comment:
Should we just `import "github.com/eug48/fhir/models"` for this stuff rather than duplicate?